src/api/index.tsx

```diff
@@ -0,0 +1,36 @@
+import axios from "axios";
+import { User } from "../types";
+
+export const fetchUsers = async (): Promise<string[]> => {
+  try {
+    const response = await axios.get<User[]>(
+      "https://jsonplaceholder.typicode.com/users"
+    );
+    return response.data.map((user) => user.name.split(" ")[0]);
```

Comment:
There's one catch with this api. Some user names start with e.g. `Mrs.`, which breaks the code if it's not addressed.